src/tags/object/RichText/table.js

```diff
@@ -19,12 +19,26 @@
 import React from 'react';
 import { inject, observer } from 'mobx-react';
 import { types } from 'mobx-state-tree';
+import { MathJaxContext, MathJax } from 'better-react-mathjax';
 import { cn } from '../../../utils/bem';
 
 import './RichText.styl';
 import { RichTextModel } from './model';
 import { HtxRichText } from './view';
 
+// Note: We use a different marker than what is used in Khanmigo.
+const MAJX_MARKER = '$';
+
+// Extract math from conversation
+// Khanmigo uses "\(.*?\)" as the marker for math
+// TODO: only extract first match at the moment
```

Comment:
I should be able to finish this TODO with this PR - looks like there are a few cases that we "do" want to display multiple expressions.